osmaxx/api_client/API_client.py

```diff
@@ -4,6 +4,8 @@
 
 logger = logging.getLogger(__name__)
 
+HTTPError = requests.HTTPError
```

Comment:
Because that would lead to
> Running flake8 code linting
> osmaxx/api_client/API_client.py:4:1: F401 'HTTPError' imported but unused
> flake8 failed